### ISSUE TITLE
"Message" wording on the app

### ISSUE THREAD

**Issue by samtaggart** [threefoldtech/threefold_wallet#237]
I think we should change _Message_ to _Message (Memo)_ or just _Memo_.

Sometimes, Memo is required and important (e.g. when sending tokens to a third-party wallet or exchange) and can mess up the transaction if the Memo is not done correctly. To avoid confusion, I recommend this change.

**Comment by ken-de-moor** [comment]
@JonasWijne easy change to include in next update

**Comment by JonasWijne** [comment]
@ken-de-moor,

While i do agree that this is an easy fix, i must note that the stellar network has diffrent types of memo's (id, text, hash, etc... See https://developers.stellar.org/docs/glossary/transactions/#memo) 
Currently we only use memo text, so changing the text to memo would imply we fully support memo, which isn't the case at this moment, to do that we would need an advanced feature to toggle the memo type

Tldr: i will change the text but we should implement different types at a later stage

**Comment by sasha-astiadi** [comment]
whatever we call in stellar we should call it the same on tf wallet
